suggestions/file_managers.json

```diff
@@ -4,5 +4,12 @@
         "label": "Material Files",
         "source": "fg",
         "repo": "https://github.com/zhanghai/MaterialFiles"
+    },
+    {
+        "id": "com.simplemobiletools.filemanager.pro",
+        "label": "Simple File Manager Pro",
+        "source": "fg",
+        "reason": "Handy file explorer for easily searching or organizing your files and folders",
+        "repo": "https://github.com/SimpleMobileTools/Simple-File-Manager"
```

Comment:
Simple File Manager is nowhere near Material Files in terms of both UI and UX.

Reply:
I agree with you 😅

✅ Will be removed in the next commits